services/ui-src/.env.tpl

```
@@ -0,0 +1,23 @@
+LOCAL_LOGIN=true
+API_REGION=op://mdct_devs/carts_secrets/AWS_DEFAULT_REGION
+API_URL=http://localhost:3030/local
+COGNITO_REGION=op://mdct_devs/carts_secrets/AWS_DEFAULT_REGION
+COGNITO_IDENTITY_POOL_ID=op://mdct_devs/carts_secrets/COGNITO_IDENTITY_POOL_ID
+COGNITO_USER_POOL_ID=op://mdct_devs/carts_secrets/COGNITO_USER_POOL_ID
+COGNITO_USER_POOL_CLIENT_ID=op://mdct_devs/carts_secrets/COGNITO_USER_POOL_CLIENT_ID
+COGNITO_USER_POOL_CLIENT_DOMAIN=op://mdct_devs/carts_secrets/COGNITO_USER_POOL_CLIENT_DOMAIN
+COGNITO_REDIRECT_SIGNIN=http://localhost:3000/
+COGNITO_REDIRECT_SIGNOUT=http://localhost:3000/
+POST_SIGNOUT_REDIRECT=op://mdct_devs/carts_secrets/POST_SIGNOUT_REDIRECT
+S3_ATTACHMENTS_BUCKET_REGION=op://mdct_devs/carts_secrets/AWS_DEFAULT_REGION
+S3_ATTACHMENTS_BUCKET_NAME=op://mdct_devs/carts_secrets/S3_ATTACHMENTS_BUCKET_NAME
+S3_LOCAL_ENDPOINT=http://localhost:4569
+BRANCH_NAME=placeholder
+API_POSTGRES_URL=https//default.dev.api_postgres.com
+REACT_APP_LD_SDK_CLIENT=op://mdct_devs/carts_secrets/REACT_APP_LD_SDK_CLIENT # pragma: allowlist secret
+LD_PROJECT_KEY=op://mdct_devs/carts_secrets/LD_PROJECT_KEY
+STAGE=local
+API_SQLSERVER_URL=https//default.dev.api_sqlserver.com
+OIDC_CLIENT_ID=op://mdct_devs/carts_secrets/OIDC_CLIENT_ID
+OIDC_ISSUER=op://mdct_devs/carts_secrets/OIDC_ISSUER
+OIDC_ISSUER_URL=op://mdct_devs/carts_secrets/OIDC_ISSUER_URL
```

I think these can be removed and we can remove [this API_POSTGRES_URL reference](https://github.com/Enterprise-CMCS/macpro-mdct-carts/blob/main/services/ui-src/src/setupTests.js#L17-L19) as well
```suggestion
REACT_APP_LD_SDK_CLIENT=op://mdct_devs/carts_secrets/REACT_APP_LD_SDK_CLIENT # pragma: allowlist secret
LD_PROJECT_KEY=op://mdct_devs/carts_secrets/LD_PROJECT_KEY
STAGE=local
```

---

This is correct, the OIDC ones are a relic of an attempted migration that got shelved. The Postgres and Sql ones can go, they're old carts

---

Correct